src/test/resources/features/Login27.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Login feature
+
+  @smoke @regression
+  Scenario: Login Success
+    Given I open Login Page
+    When I enter email "demo@class.com"
+    And I enter password "te$t$tudent"
+    And I submit
+    Then I am logged in
```

Comment:
Tests failed

![image](https://user-images.githubusercontent.com/104761482/215726374-c9dcc37a-30c0-489a-90e2-700bad6c1b26.png)

Before All/After All failed
io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in stepDefinition.Login27StepDef.iEnterEmail(java.lang.String) and stepDefinition.LoginStepDefinition.iEnterEmail(java.lang.String)